### PR TITLE
Create Event tracking guideline

### PR DESCRIPTION
**Pull request by konopkja**
Event tracking guideline

## Description

For developers to see how to structure event tracking and what to measure when creating/redesigning a page.

### CODE REVIEW

File: docs/event-tracking.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - eventName: WalletName
+  - Value: position
+
+Such division allows to specifically identify where user clicked on the page, if all external links were under one category "ExternalLink", we would not be able to measure the performance difference between the list and the modal window.
+
+## Hidden gem of tracking: Value
+
+Can be used to get more info on the UX.
+
+Examples:
```
Comment (samajammin) [inline docs/event-tracking.md:52]: Note to self: let's add a code example e.g. onClick event handler to demonstrate the util function.

**Review by samajammin — COMMENTED** [review]
FYI I renamed this to be a markdown file. Will review today.

File: docs/event-tracking.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+# How to name events?
+
+Broadly, events should be grouped by specific topic (e.g. L2 page external links, selected bridge, selected cex).
+
+## Each event comprises of 4 hierarchical values:
+
+1. Category (other events may share the same category if one feature has several actions)
+2. Action
+3. Label (optional)
```
Comment (minimalsm) [inline docs/event-tracking.md:44]:
```suggestion
3. Name (optional)
```

File: docs/event-tracking.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+4. Value (optional, can be number or text)
+
+## Category
+
+Please consider the visual position on the page, when deciding on which events should be grouped together (under the same category). Ideally, all events related to one feature should be grouped together under the same category only if there is also a single position on the page where the event can be triggered. If in doubt, always use different category when a feature is repeated on the page.
+
+Example:
```
[the remark this answers is not in the document]
Comment (minimalsm) [reply docs/event-tracking.md:51]: @samajammin [added a quick example here](https://github.com/ethereum/ethereum-org-website/pull/6630/commits/22c8b118964593bd3d9b6ec928ac0fe106ab5524), is this what you were looking for?

**Review by minimalsm — APPROVED** [review]
LGTM :-) Nice work @konopkja

File: docs/event-tracking.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (minimalsm) [reply docs/event-tracking.md]: Fixed: https://github.com/ethereum/ethereum-org-website/pull/6630/commits/9ddd7ad45724b22a69e201f00bd6900b0b0efbcd

**Pull request merged**